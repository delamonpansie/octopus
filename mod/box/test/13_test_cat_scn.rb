#!/usr/bin/ruby

$: << File.dirname($0) + '/lib'
require 'run_env'

RunEnv.connect_eval do
  16.times {|i| insert [i,i]}

  open('|./octopus --cat 10 2>/dev/null').each_line do |l|
    puts l.gsub(/tm:\d+\.\d+ /, '')
  end
end
