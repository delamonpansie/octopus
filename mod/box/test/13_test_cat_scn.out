# box.insert([0, 0])
1

# box.insert([1, 1])
1

# box.insert([2, 2])
1

# box.insert([3, 3])
1

# box.insert([4, 4])
1

# box.insert([5, 5])
1

# box.insert([6, 6])
1

# box.insert([7, 7])
1

# box.insert([8, 8])
1

# box.insert([9, 9])
1

# box.insert([10, 10])
1

# box.insert([11, 11])
1

# box.insert([12, 12])
1

# box.insert([13, 13])
1

# box.insert([14, 14])
1

# box.insert([15, 15])
1

lsn:1 shard:0 scn:1 t:snap/snap_initial count:0 run_crc_log:0x00000000 run_crc_mod:0x00000000
lsn:1 shard:0 scn:1 t:snap/snap_final 
lsn:2 shard:0 scn:2 t:wal/usr13 INSERT n:0 flags:00000000 <0:"\x00\x00\x00\x00", 0:"\x00\x00\x00\x00">
lsn:3 shard:0 scn:3 t:wal/usr13 INSERT n:0 flags:00000000 <1:"\x01\x00\x00\x00", 1:"\x01\x00\x00\x00">
lsn:4 shard:0 scn:4 t:wal/usr13 INSERT n:0 flags:00000000 <2:"\x02\x00\x00\x00", 2:"\x02\x00\x00\x00">
lsn:5 shard:0 scn:5 t:wal/usr13 INSERT n:0 flags:00000000 <3:"\x03\x00\x00\x00", 3:"\x03\x00\x00\x00">
lsn:6 shard:0 scn:6 t:wal/usr13 INSERT n:0 flags:00000000 <4:"\x04\x00\x00\x00", 4:"\x04\x00\x00\x00">
lsn:7 shard:0 scn:7 t:wal/usr13 INSERT n:0 flags:00000000 <5:"\x05\x00\x00\x00", 5:"\x05\x00\x00\x00">
lsn:8 shard:0 scn:8 t:wal/usr13 INSERT n:0 flags:00000000 <6:"\x06\x00\x00\x00", 6:"\x06\x00\x00\x00">
lsn:9 shard:0 scn:9 t:wal/usr13 INSERT n:0 flags:00000000 <7:"\x07\x00\x00\x00", 7:"\x07\x00\x00\x00">
lsn:10 shard:0 scn:10 t:wal/usr13 INSERT n:0 flags:00000000 <8:"\x08\x00\x00\x00", 8:"\x08\x00\x00\x00">
